src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'grasp')
+def fuze_grasp(robot, fuze, manip=None):
```

Comment:
The TSRFactory function just returns the list of TSRChains that can be used to successfully complete a grasp. So this function is actually used by the PushGrasp action. In PushGrasp we call ```robot.tsrlibrary(object, 'grasp')```  to get the TSR to plan to. We hadn't yet defined any TSRs for the fuze bottle.